### ISSUE TITLE
Description of the app flow for Figma

### ISSUE THREAD

**Issue by almahdc** [almahdc/daido-project-onboarding#36]
First draft:

The platform is intented as a B2B platform. Home V2 is the landing page - the product that we are offering is for onboarders, which will then invite onboardees to take a survey in order to get the whole process of onboarding companies (manufacturers) to a responsible supply chain. 

Onboarder needs to onboard the whole supply chain to this responsible supply chain and the incentive comes from down stream - from OEMs. Currently we don’t show that info in the platform, neither to what part of supply chain (midstream, upstream) does the onboardee belong. 

After the onboarder is interested in the product they schedule a call with us (Sign in - Onboarder & Onboardee -> button Don’t have an account?). All of the details of setting up the onboarder’s account are handeled in the background. 

When the onboarder has the account they can send invitations to onboardees but only after they already created the survey (as there is no need to invite onboardees without anything for them to do). 

Onboarder will see on their dashboard, overview of their onboardees, created surveys and each collaborator would see their own tasks -> which sections (questions) to create (as adding collaborators when creating a survey would also be an option - since there are multiple different part e.g. company questions, process flow questions -> if this complicates things - REMOVE IT).

Onboarder has multiple types of questions to choose from when creating a survey and can save a survey without sending it out to onboardee right away (withough assigning it to onboardee right away) - those surveys go to the section Templates and can be chosen from later when creating a new survey (and can be edited - Onboarder - Survey creation - From already predefined survey). 

Onboarded has insight in the progress of the onboardee, a pretty detailed insight. Onboarder has also insight in onboardee’s project plan (DOES THIS MAKE SENSE? OR WAS THIS IMAGINED TO BE ONBOARDER’S PROJECT PLAN ???). Onboarder can in any moment choose to export already done survey (some or all questions answered) in multiple export forms (to be elaborated later - when we get more info about this). 

When oboarder sends the survey to an oboardee, the onboardee would recive an email to accept an invitation to Kollect platform. After creating their profile (Accept invitation), the onboardee would be presented with short intro to what they got themselves into (Onboardee - Dashboard - Start). Onboardees can see the deadline (and be able to create project plan), attached documents about the onboarding process, the summary of the survey (in order to know what documents they would need). 

When the onboardee starts the survey they can assign any question to anybody they invite to the platform. When the onboardee comes to the process flow section, they would be able to edit the process flow which will be created automatically based on their answers to questions in the process flow section.
